nih-sparc/sparc-design-system-components
# Icon component

# Description

The purpose of this PR is to add the icon component and documentation.

## Type of change


# How Has This Been Tested?

- Go to the [icons](http://localhost:6006/?path=/docs/components-icons--background) documentation page. Icons should be present and documentation should outline usage. _Note: The icons are different than the designs because the "Design System only shows an example of some of them."_
- Go to the [buttons icons page](http://localhost:6006/?path=/docs/components-button--icons) and the icons should be shown in the circle buttons.

# Checklist:
